src/Query/Compound/FunctionScoreQuery.php

```diff
@@ -53,13 +53,14 @@ public function __construct(BuilderInterface $query, array $parameters = [])
      *
      * @return $this
      */
-    public function addFieldValueFactorFunction($field, $factor, $modifier = 'none', BuilderInterface $query = null)
+    public function addFieldValueFactorFunction($field, $factor, $modifier = 'none', $missing = null, BuilderInterface $query = null)
     {
         $function = [
             'field_value_factor' => [
                 'field' => $field,
                 'factor' => $factor,
                 'modifier' => $modifier,
+                'missing' => $missing
```

Comment:
Here' to keep the BC would be nice to add the whole array to the `array_filter()`.